BuildAssets/test-linux-packages.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script requires environment variables to run:
+#   OCTOPUS_CLI_SERVER, OCTOPUS_CLI_API_KEY, OCTOPUS_SPACE, OCTOPUS_EXPECT_ENV
+# You must also supply either:
+#   TEST_QUICK (to skip some distributions)
+# or
+#   REDHAT_SUBSCRIPTION_USERNAME, REDHAT_SUBSCRIPTION_PASSWORD (to support RHEL tests)
+
+TEST_DEB_SH='
+  # Configure apt
+  export DEBIAN_FRONTEND=noninteractive
+  apt-get update --quiet 2 || exit
+  apt-get install --no-install-recommends --yes apt-utils >/dev/null 2>&1 || exit # silence debconf warnings
+
+  # Install octo
+  dpkg -i /pkgs/octopuscli*.deb >/dev/null 2>&1 # Silenced and expected to fail
```

Comment:
can you expand why it's expected to fail?

Reply:
Done.